tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/metadata.py

```diff
@@ -56,6 +57,20 @@ def validate(ctx: click.Context):
     )
 
 
+@metadata.command(cls=DaggerPipelineCommand, help="Deploy the metadata service orchestrator to production")
+@click.pass_context
+def deploy_orchestrator(ctx: click.Context):
```

Comment:
What do you think about creating an `orchestrator` command group to have a command like
`airbyte-ci metadata deploy orchestrator` 

If we decide to release the lib I can easily see something like this coming
`airbyte-ci metadata deploy lib`

Reply:
Great ideas!